api_backend.yaml

```diff
@@ -111,6 +112,7 @@ paths:
             "$ref": "#/definitions/ServicePreference"
           examples:
             application/json:
+              is_allowed_send_read_message_status: true
```

Comment:
just a question about the naming: why the `send` part? maybe something like
`can_access_message_read_status` fits better (or `is_message_read_status_allowed`)

Reply:
+1

plus, _can do_ over _is allowed to do_ imho

Reply:
Thanks, i like `can_access_message_read_status`. I've added the `send` part because the read status is always collected and the preference is related to the capability of the sender to read this status, but your suggestion fits better :D